components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/resources/store-api.yaml

```diff
@@ -234,6 +234,62 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+
+  /apis/{apiId}/generate-sdk/{language}:
+#-----------------------------------------------------
+# Generate an SDK (System Development Kit) for an API in a requested language
+#-----------------------------------------------------
+    post:
+      produces:
+        - application/zip
+      x-scope: apim:subscribe
```

Comment:
please remove x-scopes and use Scope definition coming from swagger. Refer other resources in store-api.yaml